allensdk/brain_observatory/ecephys/optotagging_table/README.md

```diff
@@ -0,0 +1,30 @@
+Optotagging table
+=================
+
+Compiles a table of information about the optotagging stimulation on this experiment. This table has the following columns:
+- Start: the onset time (global clock) of optical stimulation
+- condition: integer identifier for the optical stimulation pattern
+- level: intensity (in volts output to the LED) of stimulation
+
+The conditions are:
+- 0 = 2.5 ms pulses at 10 Hz for 1 s
+- 1 = 5 ms pulse
+- 2 = 10 ms pulse
+- 3 = 1 s raised cosine pulse
+
+See the example gallery for a plot of the conditions.
+
+
+Input data
+----------
+- Optotagging pickle : a pickled Python dictionary containing 4 keys:
```

Comment:
eww... this can't be a yml/json?

Reply:
it would be really nice, but the pickles are what we get from the rigs. Same with stimulus information. Yes, this has caused problems.